uma_vasp/demo/demo_user_service.py

```diff
@@ -7,7 +7,7 @@
 from uma_vasp.user import User
 from uma_vasp.user_service import IUserService
 
-_user_name = os.environ.get("LIGHTSPARK_UMA_RECEIVER_USER") or "alice"
+_user_name = os.environ.get("LIGHTSPARK_UMA_SENDER_USER") or "alice"
```

Comment:
This isn't a safe change to make because it'll break the integration tests and our local configs at the moment. Technically this user name is both the sender and receiver, so we could just remove the name "SENDER" or "RECEIVER", but we'll also need to make sure we update test configs.

Reply:
Got it, removed this change for now.